frontend/src/pages/moments/momentsListView.tsx

```diff
@@ -5,47 +5,125 @@ import {
   IonLabel,
   IonButton,
   IonListHeader,
-  IonList,
+  IonText,
   IonRouterLink,
+  IonItemDivider,
 } from '@ionic/react'
 import { Header, MomentCard } from '../../components'
 import { RouteComponentProps } from 'react-router'
 
+import { API, graphqlOperation } from 'aws-amplify'
+import { listMoments } from '../../graphql/queries'
+
+import {
+  TextVariantWithProps,
+  ImageVariantWithProps,
+  AudioVariantWithProps,
+  VideoVariantWithProps,
+} from '../../components/card/moments/basicCard'
+import React from 'react'
+
 interface Props extends RouteComponentProps<{}> {}
 
+const data: {
```

Comment:
I thought you would remove this development placeholder?

Reply:
This code are removed with the lates commit.